main.tf

```diff
@@ -55,6 +55,27 @@ resource "azurerm_storage_account" "storage_account" {
   )
 }
 
+resource "azurerm_storage_management_policy" "storage-account-policy" {
```

Comment:
would it be worth adding an example to the README?

Reply:
Added example, documentation and comments